Nef_3/include/CGAL/Nef_polyhedron_3.h

```diff
@@ -49,7 +49,7 @@
 #include <CGAL/Nef_3/Mark_bounded_volumes.h>
 
 #include <CGAL/IO/Verbose_ostream.h>
-#include <CGAL/Nef_3/polyhedron_3_to_nef_3.h>
+#include <CGAL/Nef_3/polygon_mesh_to_nef_3.h>
```

Comment:
This include will crete a new hard dependency to CGAL BGL package. Do we want that?

As the BGL package is LGPL, that is probably not really an issue. But for non-GeometryFactory developers, that BGL package probably seems like the Algebraic Foundation of MPII: it is more and more an artificial dependency of a lot of CGAL packages.